web/src/components/molecules/Integration/IntegrationTable/index.tsx

```diff
@@ -3,75 +3,125 @@ import styled from "@emotion/styled";
 import Button from "@reearth-cms/components/atoms/Button";
 import ConfigProvider from "@reearth-cms/components/atoms/ConfigProvider";
 import Icon from "@reearth-cms/components/atoms/Icon";
-import ProTable, { ListToolBarProps } from "@reearth-cms/components/atoms/ProTable";
+import PageHeader from "@reearth-cms/components/atoms/PageHeader";
+import ProTable, { ListToolBarProps, ProColumns } from "@reearth-cms/components/atoms/ProTable";
+import Switch from "@reearth-cms/components/atoms/Switch";
+import { IntegrationMember } from "@reearth-cms/components/molecules/Integration/types";
 import { useT } from "@reearth-cms/i18n";
 
-const IntegrationTable: React.FC = () => {
-  const dataSource: [] = [];
+export type Props = {
+  integrationMembers: IntegrationMember[];
+  onIntegrationConnectModalOpen: () => void;
+  onSearchTerm: (term?: string) => void;
+  onIntegrationSettingsModalOpen: (integrationMember: IntegrationMember) => void;
+};
+
+const IntegrationTable: React.FC<Props> = ({
+  integrationMembers,
+  onIntegrationConnectModalOpen,
+  onSearchTerm,
+  onIntegrationSettingsModalOpen,
+}) => {
   const t = useT();
 
-  const columns = [
+  const columns: ProColumns<IntegrationMember>[] = [
     {
       title: t("Name"),
-      dataIndex: "name",
+      dataIndex: ["integration", "name"],
       key: "name",
       filters: [],
     },
+    {
+      title: t("State"),
+      key: "state",
+      render: (_, integrationMember) => (
+        <Switch
+          checked={integrationMember.active}
+          checkedChildren={t("ON")}
+          unCheckedChildren={t("OFF")}
+        />
+      ),
+    },
     {
       title: t("Role"),
-      dataIndex: "role",
+      dataIndex: "integrationRole",
       key: "role",
     },
     {
       title: t("Creator"),
-      dataIndex: "creator",
+      dataIndex: ["integration", "developer", "name"],
       key: "creator",
     },
     {
-      title: t("State"),
-      dataIndex: "state",
-      key: "state",
-    },
-    {
-      title: t("Action"),
-      dataIndex: "action",
       key: "action",
+      render: (_, integrationMember) => (
+        <Icon
+          style={{ color: "#1890FF", fontSize: "18px" }}
+          onClick={() => onIntegrationSettingsModalOpen(integrationMember)}
+          icon="settings"
+        />
+      ),
     },
   ];
 
   const handleToolbarEvents: ListToolBarProps | undefined = {
     search: {
-      onSearch: _ => {},
+      onSearch: (value: string) => {
+        if (value) {
+          onSearchTerm(value);
+        } else {
+          onSearchTerm();
+        }
+      },
```

Comment:
value is of type string so it'll never reach the else statement.

Reply:
@KaWaite , my fault I have copied this from the Assets search without much thinking about the statement.
Fixed, and tested as well.

Reply:
https://github.com/reearth/reearth-cms/blob/58c8be6313ec45a489b63534dab11973f18ce352/web/src/components/molecules/Integration/IntegrationTable/index.tsx#L69-L71